codes/ruby/chapter_tree/binary_tree_dfs.rb

```diff
@@ -0,0 +1,62 @@
+=begin
+File: binary_tree_dfs.rb
+Created Time: 2024-04-18
+Author: Xuan Khoa Tu Nguyen (ngxktuzkai2000@gmail.com)
+=end
+
+require_relative '../utils/tree_node'
+require_relative '../utils/print_util'
+
+### 前序遍历 ###
+def pre_oder(root)
```

Comment:
`pre_oder` -> `pre_order`

I've fixed it

Reply:
Oh! Thanks for spotting it out.